OrbitGl/TimeGraph.cpp

```diff
@@ -568,16 +568,14 @@ void TimeGraph::Draw(bool a_Picking) {
 void TimeGraph::DrawTracks(bool a_Picking) {
   m_Layout.SetNumCores(GetNumCores());
   for (auto& track : sorted_tracks_) {
-    if (track->GetLabel().empty()) {
-      if (track->GetType() == Track::kThreadTrack) {
-        auto thread_track = std::dynamic_pointer_cast<ThreadTrack>(track);
-        uint32_t tid = thread_track->GetThreadId();
-        std::string thread_name =
-            Capture::GTargetProcess->GetThreadNameFromTID(tid);
-        track->SetName(thread_name);
-        std::string track_label = absl::StrFormat("%s [%u]", thread_name, tid);
-        track->SetLabel(track_label);
-      }
+    if (track->GetType() == Track::kThreadTrack) {
+      auto thread_track = std::dynamic_pointer_cast<ThreadTrack>(track);
```

Comment:
this is not checked... so I suppose you can just use static_cast here... or check the result of dynamic cast.

Reply:
I changed it to `static_pointer_cast`. The "pointer" part is because it's used on a `shared_ptr`.